tools/cli/commands/changelog.js

```diff
@@ -766,15 +777,22 @@ async function promptChangelog( argv, needChangelog, types ) {
 		],
 	} );
 
-	// Get the type of change.
-	const { type } = await prompt( {
-		type: 'autocomplete',
-		name: 'type',
-		message: 'Type of change.',
-		suggest: ( input, choices ) => choices.filter( choice => choice.value.startsWith( input ) ),
-		highlight: v => v,
-		choices: typeChoices,
-	} );
+	// Get the type, set it to other if this isn't a user facing change.
+	let typeResponse;
+	if ( ! userFacing.userFacing && argv.project === 'plugins/jetpack' ) {
```

Comment:
Same as above regarding `argv.project`. But really we should only need to check the `userFacing` answer without checking again for plugins/jetpack at all.